bannou/client.py

```diff
@@ -2,12 +2,13 @@
 
 import pathlib
 
-import databases
 import hikari
+import orjson
 import tanjun
+from sqlalchemy.ext import asyncio as sqlalchemy_async
 
+from bannou import database as db
```

Comment:
```suggestion
from bannou import database
```

Or why not just name the module `db`

Reply:
The module name is for clarity, the shortened alias import is just to make it easier to type 😝